templates/includes/npm.php

```diff
@@ -184,13 +184,13 @@
 <h4>Using ES6 modules natively.</h4>
 
 <p>
-  This is the method that this page you are reading now loads the scripts.
+  This is the method by which this page you are reading now loads the scripts.
```

Comment:
I think the following makes more sense:

```suggestion
  This is the method by which the page you are reading now loads the scripts.
```